infra-envs.yml

```diff
@@ -0,0 +1,15 @@
+- name: Test infra-envs
+  hosts: localhost
+  tasks:
+
+    - name: Create a infra-env
+      infra-envs:
+        state: present
+        name: testinfra
+        pull_secret: '<insert your pullsecret between the single quotes>'
```

Comment:
Per the tests this needs to not be logged

Reply:
Thanks @dkosteck

Reply:
FYI I learnt something new here from this feedback